syn/binder.go

```diff
@@ -33,6 +33,10 @@ func (n Name) VarEncode(e *encoder) error {
 		return err
 	}
 
+	if n.Unique > math.MaxUint {
```

Comment:
`math.MaxUint` will have a different value on 32-bit vs. 64-bit platforms. We should probably use a fixed-size type instead